course7/homeworks/fred/complex-rpn/main.cc

```diff
@@ -0,0 +1,65 @@
+#include "complex.h"
+#include "calculator.h"
+#include "utils.h"
+
+#include <iostream>
+#include <stdexcept>
+
+void insert(RpnCalculator& calculator, Complex complex) {
```

Comment:
What is the value added by this function?

Reply:
I just wanted to maintain a standard